### PR TITLE
[#127] Add more details about raw and semantic tokens definitions in documentation

### PR DESCRIPTION
**Pull request by pylapp**
_Note: Please transform `- [ ]` into `- (NA)` in the description when things are not applicable_

### Related issues

#127

### Description

Add more details about how to define, add, update or remove raw and semantic tokens

### Types of change

- Refactoring (non-breaking change)

#### Contribution

- [x] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/ouds-ios/blob/develop/.github/CONTRIBUTING.md)

#### Accessibility

- (NA) My change follows accessibility good practices

#### Design

- (NA) My change respects the design guidelines of _Orange Unified Design System_

#### Development

- [x] My change follows the [developer guide](https://github.com/Orange-OpenSource/ouds-ios/blob/develop/.github/DEVELOP.md)
- [ ]  I have added unit tests to cover my changes _(optional)_

#### Documentation

- [x] My change introduces changes to the documentation and/or I have updated the documentation accordingly

### Checklist (for Core Team only)

- [x] The evolution have been tested and the project builds for iPhones and iPads
- [x] Code review has been done by reviewers according to [CODEOWNERS file](https://github.com/Orange-OpenSource/ouds-ios/blob/develop/.github/CODEOWNERS)
- (NA) Design review has been done
- (NA) Accessibiltiy review has been done
- (NA) Q/A team has tested the evolution
- [x] Documentation has been updated if relevant
- [x] Internal files have been updated if relevant (like CONTRIBUTING, DEVELOP, THIRD_PARTY, CONTRIBUTORS, NOTICE)
- [x] CHANGELOG has been updated respecting [keep a changelog rules](https://keepachangelog.com/en/1.0.0/) and reference the issues

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [Library] Update color semantic tokens to better manage light and dark modes values
 - [Library] Update elevation colors focus light and dark
 - [Library] Rename space padding inline component tokens by removing "component" word
+- [Library] Improve documentation about raw tokens definitions
```
Comment (Tayebsed93) [inline CHANGELOG.md:24]: je me permets de poser une question : cette ligne ne devrait-elle pas être associée à une issue ?

File: OUDS/Core/OUDS/Sources/_OUDS.docc/Tokens.md [hunk 2]
```diff
@@ -98,7 +98,7 @@ In addition, there are hundreds of _semantics tokens_ and we needed to add them
 
 That is the reason why tokens are exposed as `@objc open` to be available and overridable anywhere. 
 
-To keep the same semantics as the ones used in our specifications, _typealias_ are used to as to make the links to _primitive types_ and our logic of _tokens_. These type aliases are avaialble for those who want too make their own theme. It's only syntaxic sugar to bring _design words_ in our product.
+To keep the same semantics as the ones used in our specifications, _typealias_ are used to as to make the links to _primitive types_ and our logic of _tokens_. These type aliases are available for those who want too make their own theme. It's only syntaxic sugar to bring _design words_ in our product.
```
Comment (Tayebsed93) [inline OUDS/Core/OUDS/Sources/_OUDS.docc/Tokens.md:101]: Il y a une petite erreur de frappe : 'too' devrait être corrigé en 'to'.

### Correction :
_These type aliases are available for those who want **too** make their own theme_

File: CHANGELOG.md (hunk 1 above)
Comment (pylapp) [reply CHANGELOG.md:24]: You are right!

File: OUDS/Core/OUDS/Sources/_OUDS.docc/Tokens.md (hunk 2 above)
Comment (pylapp) [reply OUDS/Core/OUDS/Sources/_OUDS.docc/Tokens.md:101]: Well spotted!

**Comment by pylapp** [comment]
@Tayebsed93 Commit message, according to conventional commits rules, should be like

```text
docs: add details about raw and semantic tokens definitions (#127)
```

When the review is done in your side, you must fill the remaining non optional checkbox in the PR description (`Code review has been done by reviewers according to CODEOWNERS file`). Then the PR can be mergeable.

You will have to **make a merge with a squash commit** and use for example the commit message above.

**Review by Tayebsed93 — APPROVED** [review]
docs: add details about raw and semantic tokens definitions (#127)

**Pull request merged**